docs/UserGuide.md

```diff
@@ -189,4 +189,11 @@ Action | Format, Examples
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **List** | `list`
+**Group** | `group GROUP_NAME` <br> e.g., `group CS2103 Project Group`
+**Groups** | `groups`
+**Delete Group** | `deleteG g/GROUP_INDEX`<br> e.g., `deleteG g/2`
+**Adding people to a group** | `addG PERSON_INDEX_1, PERSON_INDEX_2.. g/GROUP_INDE`<br> e.g.,`addG 1, 2, 3 g/2`
```

Comment:
I think there might be a minor typo; `g/GROUP_INDE` might need to be changed to `g/GROUP_INDEX` instead.

Reply:
Fixed, thanks 😄